Files/powerGate/Modules/Communication.psm1

```diff
@@ -2,12 +2,31 @@
 Import-Module powerGate
 
 #TODO: configure the powerGate server url and port
-$powerGateServerName = $ENV:Computername
+$powerGateServerName = getRelatedPGServerName
```

Comment:
Before the default value was `$ENV:Computername` now the default value is $Null see below.